src/LLMProviders/chatModelManager.ts

```diff
@@ -96,6 +95,7 @@ export default class ChatModelManager {
       [ModelProviders.LM_STUDIO]: {
         openAIApiKey: "placeholder",
         openAIProxyBaseUrl: `${params.lmStudioBaseUrl}`,
+        useOpenAILocalProxy: params.useOpenAILocalProxy,
```

Comment:
Why do we need this for LM Studio? Is this line for testing?

Reply:
I added it here in order to allow using LM Studio with CORS disabled to test `ProxyChatOpenAI`:
<img width="479" alt="image" src="https://github.com/user-attachments/assets/85add08d-b8e9-48cc-afd0-bb93bc87056d">

Is there any other `providerConfig` that should have it enabled?